collector/main.go

```diff
@@ -3,15 +3,22 @@ package main
 import (
 	"github.com/Kindling-project/kindling/collector/application"
 	"log"
+    "github.com/Kindling-project/kindling/collector/version"
```

Comment:
Please use `gofmt` tool to format these lines.

Reply:
Correction： 
1、Standardized the format of `main.go` and print at the beginning   
2、Changed the mount point of the collector-build container
3、Separated startup and compilation. However, because the change of the mount point changes the container startup command, I don't know where to put this command temporarily, so I left this startup command in  `collector-version-build.sh`.